ui/components/app/assets/util/parseAssetId.ts

```diff
@@ -0,0 +1,63 @@
+export type ParsedAssetId = {
+  namespace: string; // Namespace (e.g., eip155, solana, bip122)
+  chainId: string; // Full chain ID (namespace + blockchain ID)
+  assetNamespace: string; // Asset namespace (e.g., slip44, erc20, token, ordinal)
+  assetReference: string; // Asset reference (on-chain address, token identifier, etc.)
+};
+
+export const parseAssetId = (assetId: string): ParsedAssetId => {
+  // Ensure input is valid
+  if (!assetId.includes('/') || !assetId.includes(':')) {
```

Comment:
Do we know how this could be the case?

Reply:
I don't, this is more of a paranoid check really to throw an error before falling into the rest of the list. I can maybe remove it if it makes sense to?